tstest/integration/vms/vms_test.go

```diff
@@ -294,6 +294,8 @@ func mkVM(t *testing.T, n int, d Distro, sshKey, hostURL, tdir string) func() {
 	t.Logf("running: qemu-system-x86_64 %s", strings.Join(args, " "))
 
 	cmd := exec.Command("qemu-system-x86_64", args...)
+	cmd.Stdout = log.Writer()
```

Comment:
let's not change this file in this commit/PR.  These are two unrelated things. They should have their own commits.